### PR TITLE
Correcting gherkin script 

### PR DESCRIPTION
**Pull request by sanketpathak**
Task: https://issues.redhat.com/browse/ODC-5753

### CODE REVIEW

File: frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-secrets.feature [hunk 1]
```diff
@@ -9,14 +9,17 @@ Feature: Secrets
 
         @smoke
         Scenario: Add Secrets: P-08-TC01
```
Comment (jerolimov) [inline frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-secrets.feature:11]:
```suggestion
        Scenario Outline: Add Secrets: P-08-TC01
```

Scenario Outline is required when using a examples table, right?

Comment (sanketpathak) [reply frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-secrets.feature:11]: Great catch, thanks

Comment (sanketpathak) [reply frontend/packages/pipelines-plugin/integration-tests/features/pipelines/pipelines-secrets.feature:11]: Updated

**Review by jerolimov — APPROVED** [review]

**Pull request merged**